### PR TITLE
Merge CustomFolderIcons with Customization properties page

### LINKED ISSUE #7431: Merge CustomFolderIcons with Customization properties page
### What's the Problem?

_No response_

### Solution/Idea

Merge CustomFolderIcons with Customization properties page to reduce duplication

### Alternatives

NA

### Priorities

_No response_

### Files Version

_No response_

### Windows Version

_No response_

### Comments

_No response_

### PR DESCRIPTION
**Pull request by 0x5bfa**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #7431

**Changes**

- [x] Merge CustomFolderIcons with Customization properties page
- [x] Add ViewModel of Customization page

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Comment by yaira2** [comment]
> Tweaked UI of Customization page

This wasn't on the list, I'll review the changes but make sure to open an issue and get approval in the future.

File: src/Files.App/Views/CustomFolderIcons.xaml
Comment (yaira2) [inline src/Files.App/Views/CustomFolderIcons.xaml:2]: This view can be removed

**Comment by yaira2** [comment]
> Add ViewModel of Customization page

Do you want to do that in this PR?

**Comment by onein528** [comment]
I'm doing that.

**Comment by onein528** [comment]
> > Tweaked UI of Customization page
> 
> This wasn't on the list, I'll review the changes but make sure to open an issue and get approval in the future.
> 
> What changes were made?

I have should written like: Fixed margin of GridView

**Comment by yaira2** [comment]
This looks a lot better! Is it just the theme or is the separator line missing?

File: src/Files.App/Views/Pages/PropertiesCustomization.xaml
Comment (yaira2) [inline src/Files.App/Views/Pages/PropertiesCustomization.xaml:27]: The outer grid can be removed.

**Comment by onein528** [comment]
> This looks a lot better! Is it just the theme or is the separator line missing?

Yeah, But you know, Main property page hosting NavigationView and OK/Cancel buttons has ScrollViewer wrapping Frame. So I set MaxHeight and if you change window height:

To fix this, remove ScrollViewer from Main property page 
image

**Comment by yaira2** [comment]
> I changed color with: (whole separator should be this, I believe. But what do you think?)

Let's restore to what it was before, it's not visible like this

**Comment by yaira2** [comment]
> To fix this, I need to remove ScrollViewer from Main property page

Let's hold off on this change, I have some redesign work for the properties window (let me know if you're interested). I'd prefer if we limited this PR to code quality.

**Comment by 0x5bfa** [comment]
Done. @yair100 

No UI changed

image

**Review by yair100 — APPROVED** [review]
LGTM

Love the use of commands and the new view model!

**Pull request merged**
